src/file.cr

```diff
@@ -377,6 +378,15 @@ class File < IO::FileDescriptor
     stat.size
   end
 
+  # Truncates the file to the specified length. Requires a write file descriptor
+  def truncate(length)
```

Comment:
@asterite we were discussing an issue in this outdated comment thread here, sorry :P